chocolatey-community/chocolatey-packages
# (blender) Blender 3 upgrade

## Description
This changes the way blender is distributed. Instead of packing the EXE it now downloads it as part of the installation process.

## Motivation and Context
The new version of blender is to big to be uploaded to the community package repo (see ticket #1763). This changes the way blender is installed.

## How Has this Been Tested?

1. Run `./update.ps1` to ensure auto update works with all new parameters
2. Run `choco pack` to ensure it still packs properly
3. Run `choco update blender -s .` from package repo to ensure it can actually install (or in my case update)

Also did run `Test-Package`

## Screenshot (if appropriate, usually isn't needed):

## Types of changes


## Checklist:
